src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs

```diff
@@ -485,14 +486,18 @@ internal static string AppendChangesToDescription(string prDescription, Build ol
 
             var description = new StringBuilder(prDescription + Environment.NewLine);
 
-            var firstCommit = changes[0];
             var repoURL = $"http://github.com/{oldBuild.Repository.Id}";
 
             foreach (var commit in changes)
             {
-                // Exclude auto-merges
-                if (commit.Author == "dotnet-automerge-bot" ||
-                    commit.Author == "msftbot[bot]")
+                // Exclude arcade dependency updates
+                if (commit.Author == "dotnet-maestro[bot]")
```

Comment:
This was already happening by accident because the merge commit that maestro uses doesn't match the regular GitHub format. But I wanted to make it more explicit.